src/scheduled-query/azext_scheduled_query/_params.py

```diff
@@ -35,4 +35,14 @@ def load_arguments(self, _):
         c.argument('mute_actions_duration', type=get_period_type(as_timedelta=True),
                    options_list=['--mute-actions-duration', '--mad'],
                    help='Mute actions for the chosen period of time (in ISO 8601 duration format) after the alert is fired.')
-        c.argument('actions', options_list=['--action', '-a'], action=ScheduleQueryAddAction, nargs='+', validator=get_action_group_validator('actions'))
+        c.argument('actions', options_list=['--action', '-a'], action=ScheduleQueryAddAction, nargs='+',
+                   validator=get_action_group_validator('actions'), deprecate_info=c.deprecate(redirect='--actions', hide=True))
```

Comment:
You should remove `ScheduleQueryAddAction` and `get_action_group_validator` as it is deprecated.